client/state/memberships/coupon-list/test/selectors.js

```diff
@@ -0,0 +1,13 @@
+import { getCouponsForSiteId } from '../selectors';
+
+describe( 'selectors', () => {
+	describe( 'getCouponsForSiteId()', () => {
+		test( 'should return an array of coupons', () => {
```

Comment:
What about a test if we have no coupons for that site, or if the site is unknown?